share/pot/start.sh

```diff
@@ -533,19 +507,43 @@ _js_start()
 	else
 		_cmd="$( _js_get_cmd "$_pname" )"
 	fi
-	if [ -x "${POT_FS_ROOT}/jails/$_pname/conf/prestart.sh" ]; then
+	if [ -x "$_confdir/prestart.sh" ]; then
 		_info "Executing the pre-start script for the pot $_pname"
 		(
 			# shellcheck disable=SC2046
 			eval $( pot info -E -p "$_pname" )
-			"${POT_FS_ROOT}/jails/$_pname"/conf/prestart.sh
+			"$_confdir/conf/prestart.sh"
 		)
 	fi
-	_bg_start "$_pname" &
 	_info "Starting the pot $_pname"
 	# shellcheck disable=SC2086
-	jail -c -J "/tmp/${_pname}.jail.conf" $_param exec.start="$_cmd"
-	sleep 1
+	jail -c -J "/tmp/${_pname}.jail.conf" $_param exec.start="sh -c 'sleep 5&'"
+
+	if [ -e "$_confdir/pot.conf" ] && _is_pot_prunable "$_pname" ; then
+		# set-attr cannot be used for read-only attributes
+		${SED} -i '' -e "/^pot.attr.to-be-pruned=.*/d" \
+		    "$_confdir/pot.conf"
+		echo "pot.attr.to-be-pruned=YES" >> "$_confdir/pot.conf"
+	fi
+
+	if _is_pot_running "$_pname" ; then
+		_js_rss "$_pname"
+	fi
+
+	jexec "$_pname" "$_cmd" &
```

Comment:
Running some tests, I've found issues here: the correct version should be:
`jexec "$_pname" $_cmd &`
otherwise jexec will consider arguments as part of the command to be executed

Reply:
I remember having this problem when running `jexec 1 "ls -al"` in the past, so your change request makes sense.

At the same time, I tested this here with this setting in pot.conf:
```
pot.cmd=su -m postgres -c 'postgres -D /var/db/postgres/data12'
```
and it works ok.

The reason being, that I tested a norc version which writes to tinirc, so $_cmd is "/tmp/tinirc" here at that moment. Duh. Will change :)

Reply:
Don't appease shellcheck without thinking :)